lib/big_query/base/timestamp.ex

```diff
@@ -1,35 +1,56 @@
 defmodule BigQuery.Base.Timestamp do
   import BigQuery.Base.Query
 
-  def timestamp_query(tbl, where_sql, params, from_dtim, to_dtim, interval_s) do
-    timestamp_sql(tbl, where_sql)
-    |> query(timestamp_params(from_dtim, to_dtim, interval_s) |> Map.merge(params))
+  def timestamp_query(tbl, where_sql, params, interval) do
+    params
+    |> timestamp_params(interval)
+    |> query(timestamp_sql(tbl, where_sql))
```

Comment:
This syntax cleans up quite after making `params` is the first argument to `query()`, so I can just pipe it in.